### PR TITLE
v2018.1 known issues and v2018.1.1 release notes

### PR DESCRIPTION
**Pull request by NeoRaider**

### CODE REVIEW

File: docs/releases/v2018.1.1.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+* The MAC address of the WAN interface is modified even when Mesh-on-WAN is disabled (`#496 <https://github.com/freifunk-gluon/gluon/issues/496>`_)
+
+  This may lead to issues in environments where a fixed MAC address is expected (like VMware when promicious mode is disallowed).
+
+* Inconsistent respondd API (`#522 <https://github.com/freifunk-gluon/gluon/issues/522>`_)
+
+  The current API is inconsistent and will be replaced eventually. The old API will still be supported for a while.
+
+* Frequent reboots due to out-of-memory on weak hardware in larger meshes
+  (`#1243 <https://github.com/freifunk-gluon/gluon/issues/1243>`_)
```
Comment (rotanid) [inline docs/releases/v2018.1.1.rst]: do we not want to copy the known issue section from 2018.1? especially a sentence further describing the current status of the OOM stuff is missing here

Comment (NeoRaider) [reply docs/releases/v2018.1.1.rst]: Fixed.

File: docs/releases/v2018.1.rst [hunk 2]
```diff
@@ -400,3 +400,21 @@ Known issues
...
+
+* Configuration loss on upgrade under certain circumstances
+  (`#1496 <https://github.com/freifunk-gluon/gluon/issues/1496>`_)
+
+  The issue can only occur when upgrading from 2018.1 and there are multiple
+  mirror entries in *site.conf* (specifically, an early failure for one of the
+  mirrors, e.g. during DNS resolution, followed by a successful upgrade from a
+  different mirror triggers the issue).
+
+  This is a regression is Gluon 2018.1.
```
Comment (blocktrron) [inline docs/releases/v2018.1.rst]: Probably misspelled? "regression in Gluon"

File: docs/releases/v2018.1.rst [hunk 3]
```diff
@@ -400,3 +400,21 @@ Known issues
...
+
+* Next-node ARP issue
+  (`#1488 <https://github.com/freifunk-gluon/gluon/issues/1488>`_)
+
+  A routing table issue leads to ARP requests being sent from the next-node IPv4 address, but with
+  a node-specific source MAC address. This can make the next-node IPv4 address unreachable.
+
+  This is a regression is Gluon 2018.1.
```
Comment (blocktrron) [inline docs/releases/v2018.1.rst]: Same story

**Pull request merged**